utils/generic-components.stanza

```diff
@@ -173,19 +210,28 @@ public defn ceramic-cap (user-params:Tuple<KeyValue>) -> Instantiable :
   val params = remove-duplicate-keys $ [["type" => "ceramic" "min-rated-voltage" => MIN-CAP-VOLTAGE]
                                         smd-query-properties()
                                         user-params]
-  to-jitx $ Capacitor(params)
-
-public defn ceramic-cap (cap:Double) -> Instantiable :
+  val my-params = replace-params(params)
```

Comment:
`replace-params` crashes if another key than `resistance` / `inductance` / `capacitance` has a value of type `Toleranced`. You cannot make this assumption here.

Reply:
I guard against that in the updated replace-params() function.